src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/DispatchWrapperTests.cs

```diff
@@ -11,13 +11,21 @@ public class DispatchWrapperTests
     {
         [Fact]
         [PlatformSpecific(TestPlatforms.Windows)]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot, "Throws PlatformNotSupportedException in UapAot")]
         public void Ctor_NullWindows_Success()
         {
             var wrapper = new DispatchWrapper(null);
             Assert.Null(wrapper.WrappedObject);
         }
 
         [Fact]
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.UapAot, "Throws PlatformNotSupportedException in UapAot")]
+        public void Ctor_NullUapAot_ThrowsPlatformNotSupportedException()
+        {
+            Assert.Throws<PlatformNotSupportedException>(() => new DispatchWrapper(null));
+        }
```

Comment:
I think coreclr has an if block making sure this isn't null. If it is null, do nothing, If it is not, it always throws a PNSE.

Why not do this with UAP too